src/engraving/libmscore/slurtie.cpp

```diff
@@ -460,7 +460,7 @@ SlurTie::~SlurTie()
 
 void SlurTie::writeProperties(XmlWriter& xml) const
 {
-    Element::writeProperties(xml);
+    Spanner::writeProperties(xml);
```

Comment:
critical - ensures that the "ticks" property is correctly preserved when creating mimeData